pybotters/ws.py

```diff
@@ -186,7 +190,7 @@ async def _ws_connect(
         hdlr_json: list[WsJsonHandler],
         **kwargs: Any,
     ) -> None:
-        async with self._session.ws_connect(self._url, **kwargs) as ws:
+        async with self._session.ws_connect(self._url, autoping=False, **kwargs) as ws:
```

Comment:
`autoping=True` for aiohttp suppresses the reception of Pong frames. Therefore, it should be explicitly set to False.